client/src/containers/HomePage/selectors.js

```diff
@@ -0,0 +1,66 @@
+import { createSelector } from 'reselect';
+
+const selectHomePageDomain = () => state => state.get('homePageDomain');
+
+const selectDiscovered = () => createSelector(
+  selectHomePageDomain(),
+  homePageDomain => homePageDomain.get('discovered'),
+);
+
+const selectName = () => createSelector(
```

Comment:
Probably all these functions can be substituted by one... I need to take a look at it...